Makefile.am

```diff
@@ -1235,6 +1234,7 @@ libsss_iface_sync_la_LDFLAGS = \
 pkglib_LTLIBRARIES += libsss_util.la
 libsss_util_la_SOURCES = \
     src/confdb/confdb.c \
+    src/confdb/confdb_setup.c \
```

Comment:
What is the benefit to move it to 'utils'?

Reply:
The before it was in monitor, and that is not enough now. To be shared between monitor and kcm, I moved it to the library.

Reply:
I'd prefer to add .c file directly.
'utils' is too general purpose and used by all other binaries.

Reply:
Done this way.